foundation-models/openai/src/main/java/com/sap/ai/sdk/foundationmodels/openai/OpenAiClient.java

```diff
@@ -119,12 +134,13 @@ public OpenAiClient withSystemPrompt(@Nonnull final String systemPrompt) {
   }
 
   /**
-   * Generate a completion for the given user prompt.
+   * Generate a completion for the given string prompt as user.
    *
    * @param prompt a text message.
    * @return the completion output
    * @throws OpenAiClientException if the request fails
    */
+  @Deprecated(since = "1.3.0")
```

Comment:
(Minor)

Good. But there should be a section in JavaDoc `@deprecated [...]` that explains what to use instead. This way developers have a chance to find the new recommended API.